module/VuFindTheme/src/VuFindTheme/Initializer.php

```diff
@@ -440,4 +443,35 @@ protected function updateTranslator($themes)
             }
         }
     }
+
+    /**
+     * Load configured javascript files from mixin.config.php if existing
+     *
+     * @return void
+     */
+    public function loadConfiguredJavascriptFilesFromMixin(): void
+    {
+        $config = $this->serviceManager->get('config');
+        $templatePathStack = $config['view_manager']['template_path_stack'] ?? false;
+        if ($templatePathStack) {
+            /* @var HeadScript $headScript */
+            $headScript = $this->serviceManager->get('ViewHelperManager')
+                    ->get('headScript') ?? $this->serviceManager->get('headScript');
+            foreach ($templatePathStack as $templatePath) {
+                if (file_exists($mixin = $templatePath . '/../mixin.config.php')) {
```

Comment:
Is there a reason to parse the mixin configurations here? This feels like work that the ThemeInfo class should be doing, to keep that all encapsulated in the same place. Is there a way to either pull the needed information with an existing ThemeInfo method (like getMergedConfig), or should we add a new one for this purpose?